packages/storage/src/platform/node/connection.ts

```diff
@@ -21,7 +21,7 @@ import {
   ErrorCode
 } from '../../implementation/connection';
 import { internalError } from '../../implementation/error';
-import nodeFetch, { Headers } from 'node-fetch';
+import { fetch, Headers } from 'undici';
```

Comment:
Same comment as above, rename just to be totally sure there's no name collisions with Node 18+

Reply:
Done.